inference/core/models/object_detection_base.py

```diff
@@ -115,18 +124,14 @@ def make_response(
         Returns:
             List[ObjectDetectionInferenceResponse]: A list of response objects containing object detection predictions.
         """
-
         if isinstance(img_dims, dict) and "img_dims" in img_dims:
             img_dims = img_dims["img_dims"]
 
-        predictions = predictions[
-            : len(img_dims)
-        ]  # If the batch size was fixed we have empty preds at the end
+        len_img_dims = len(img_dims)
         responses = [
             ObjectDetectionInferenceResponse(
                 predictions=[
                     ObjectDetectionPrediction(
```

Comment:
```suggestion
                    ObjectDetectionPrediction(
                        # Passing args as a dictionary here since one of the args is 'class' (a protected term in Python)
```